src/main/java/se/sundsvall/casedata/service/util/Constants.java

```diff
@@ -34,6 +34,14 @@ public final class Constants {
 	public static final String NOTIFICATION_DECISION_CREATED = "Beslut skapat";
 	public static final String NOTIFICATION_DECISION_UPDATED = "Beslut uppdaterat";
 
+	public static final String NOTIFICATION_SUBTYPE_ATTACHMENT = "ATTACHMENT";
```

Comment:
Could this be an enum instead?

If there will be a requirement to return the possible values to frontend (for filtering purposes) or if we should add validation on this attribute, an enum will make this easier.